deployment/clouddeploy/gke-workers/base/generate-sitemap.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: generate-sitemap
+spec:
+  schedule: "30 8 * * *"
```

Comment:
Just to confirm, you want this to run once per day at 8:30?

Reply:
Yes, that's the idea, I didn't want to run it at midnight as it might line up with other tasks causing a spike in performance.